benchmark/utils/main.swift

```diff
@@ -68,6 +68,7 @@ import DictionaryOfAnyHashableStrings
 import DictionaryRemove
 import DictionarySubscriptDefault
 import DictionarySwap
+import Differentiation
```

Comment:
Similarly, should Swift code changes be gated by `#if canImport(_Differentiation)`?
There seems to be some precedent for flag-gated benchmarks.

Reply:
Done.